.github/workflows/docs_workflow.yml

```
@@ -36,7 +36,7 @@ jobs:
         with:
           activate-environment: "mache_ci"
           miniforge-version: latest
-          channels: conda-forge,e3sm/label/mache,defaults
+          channels: conda-forge,e3sm/label/mache
```

```suggestion
          channels: conda-forge
```
I don't think we ever use the `mache` label on the `e3sm` channel so this must just have been copied from compass or polaris.

---

Fixed!